src/lib/solana/manuallyExecuteSolana.ts

```diff
@@ -97,7 +110,26 @@ export async function buildManualExecutionTxWithSolanaDestination<
     instructions: finalInstructions,
   })
   const messageV0 = message.compileToV0Message(addressLookupTableAccounts)
-  return new VersionedTransaction(messageV0)
+  const transaction = new VersionedTransaction(messageV0)
+
+  if (transaction.serialize().length > 1232 || forceBuffer) {
+    console.log(
+      `Execute report will be pre-buffered through the buffering contract ${bufferProgramAddress}. This may take some time`,
+    )
+    return bufferedTransactions(
+      destinationProvider,
+      bufferProgramAddress,
+      serializedReport,
+      serializedTokenIndexes,
+      accounts,
+      remainingAccounts,
+      computeBudgetIx,
+      blockhash,
+      addressLookupTableAccounts,
+    )
+  } else {
+    return [transaction]
+  }
```

Comment:
Nit: since you're `return`ing inside the `if` branch, there's no need to have an `else` branch. You can just do 

```suggestion
  if (transaction.serialize().length > 1232 || forceBuffer) {
    console.log(
      `Execute report will be pre-buffered through the buffering contract ${bufferProgramAddress}. This may take some time`,
    )
    return bufferedTransactions(
      destinationProvider,
      bufferProgramAddress,
      serializedReport,
      serializedTokenIndexes,
      accounts,
      remainingAccounts,
      computeBudgetIx,
      blockhash,
      addressLookupTableAccounts,
    )
  } 
  
  return [transaction]
```

I haven't checked the linting rules configured in this repo and whether you've run the linter, but [this rule](https://eslint.org/docs/latest/rules/no-else-return) is related to this.